tools/commentMonitor/ghclient.go

```diff
@@ -65,6 +66,7 @@ func (c githubClient) createRepositoryDispatch(ctx context.Context, eventType st
 		EventType:     eventType,
 		ClientPayload: clientPayload,
 	}
+	log.Printf("creating repository_dispatch with payload: %v", clientPayload)
```

Comment:
Is there any chance that this might expose sensitive information?

Reply:
currently `client_payload` only has public information, so this should not expose any sensitive information.

eg.
```
ghclient.go:69: creating repository_dispatch with payload: map[LAST_COMMIT_SHA:00c847e79991d6613b9e38ed0b7879310ce3354b PR_NUMBER:27 RELEASE:master]
```